data/filedefs/filetypes.haskell

```diff
@@ -51,7 +51,7 @@ mime_type=text/x-haskell
 #wordchars=_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789
 
 # single comments, like # in this file
-comment_single=--
+comment_single=-- 
```

Comment:
I would rather see representing the space [as `\s`](https://developer.gnome.org/glib/unstable/glib-Key-value-file-parser.html#glib-Key-value-file-parser.description) to make it explicit we do want the space here, and avoid any possible future accidental removal by e.g. "remove trailing spaces" feature.